src/io/matpower.jl

```diff
@@ -73,7 +73,7 @@ end
 
 
 function parse_matpower(file_string)
-    data_string = readall(open(file_string))
+    @compat data_string = readstring(open(file_string))
```

Comment:
You don't need ``@compat`` here. ``readstring`` is just a method exported by the Compat module